WikiWatershed/model-my-watershed
# Micro Site fixes

## Overview

Adds Google Analytics to Micro site. While the Micro site extended from `base.html`, because the view did not populate it with the right context, the base view was unable to populate the variable.

Also updates Open Water art with new images with shading.

## Testing Instructions

 1. Pull down the branch, and run `./scripts/manage.sh collectstatic` to refresh the images.
 2. Visit the Micro site.
 3. Select "Open Water". Notice that the land cover art has a slight tint on the right side and a slight shade on the left.
 4. Open developer tools. Under `<head>`, look up the Google Analytics script. Ensure that it has a populated Google Analytics identifier.

## Demo

![image](https://cloud.githubusercontent.com/assets/1430060/10379537/bf67c8de-6ddc-11e5-99b5-0e357dd8a390.png)

Connects #907 
Connects #920
